api/rest.go

```diff
@@ -217,6 +217,29 @@ func CollectionContentUnitsHandler(c *gin.Context) {
 	concludeRequest(c, resp, err)
 }
 
+func CollectionContentUnitsPositionHandler(c *gin.Context) {
+	var err *HttpError
+	var resp interface{}
+	id, e := strconv.ParseInt(c.Param("id"), 10, 0)
+	if c.Request.Method == http.MethodPost {
```

Comment:
useless check. The router does that for you.

Reply:
fixed